plugins/ilert/src/types.ts

```diff
@@ -15,17 +15,18 @@
  */
 
 /** @public */
-export interface Incident {
+export interface Alert {
```

Comment:
First of all, thank you very much for the plenty of comments with improvements, i have addressed many of them right away.
Regarding the issue above i can say there are no additional types planned as of right now to be integrated. But if we add more types in a future update this will be addressed and refactored properly.